Tools/pynche/pynche

```diff
@@ -1,7 +1,7 @@
-#! /usr/bin/env python
+#! /usr/bin/env python3
 
 """Run this file under Unix, or when debugging under Windows.
 Run the file pynche.pyw under Windows to inhibit the console window.
 """
-import Main
```

Comment:
Could stay in the previous form, indeed other modules of pynche still us this form.

From a semantical point of view It could be preferable to focus these changes on shebang updates and refactor the pynche's imports through a follow up patch.

Thoughts?

Reply:
Example: https://github.com/python/cpython/blob/master/Tools/pynche/Main.py#L54

Reply:
You are right. And I don't know who will use it so I decide to revert this file, thanks.

Reply:
You are welcome :)